h/views/account_signup.py

```diff
@@ -91,16 +91,21 @@ def post(self):
     def validation_failure(self):
         return {
             "js_config": {
-                "formErrors": self.context.error.asdict(),
-                "formData": {
-                    "username": self.request.POST.get("username", ""),
-                    "email": self.request.POST.get("email", ""),
-                    "password": self.request.POST.get("password", ""),
-                    "privacy_accepted": self.request.POST.get("privacy_accepted", "")
-                    == "true",
-                    "comms_opt_in": self.request.POST.get("comms_opt_in", "") == "true",
-                },
                 **self.js_config,
```

Comment:
`self.js_config` is the "base" configuration here containing an empty form. It makes sense for the context-specific configuration to come afterwards.